attesters/mocktsm/mocktsm.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package mocktsm
+
+import (
+	"fmt"
+
+	"github.com/google/go-configfs-tsm/configfs/configfsi"
+	"github.com/google/go-configfs-tsm/configfs/faketsm"
+	"github.com/google/go-configfs-tsm/report"
+	"github.com/veraison/ratsd/proto/compositor"
+	"github.com/veraison/ratsd/tokens"
+)
+
+const (
+	mediaType = "application/vnd.veraison.mock-tsm+json"
```

Comment:
This should be the “usual” configfs-tsm media type.

The “provider” field in the JSON will be set to “fake” (or whatever) :-)

Reply:
Resolved.